util/src/main/java/io/kubernetes/client/informer/EventType.java

```diff
@@ -23,11 +29,15 @@
 
   ERROR;
 
```

Comment:
Every time you call `values()` on an enum, there is an inner copy created. We can pre-compute the needed `Map` before doing the search as a `static final instance` and simply do a hash lookup. `String::valueOf` is needed only to take care of `null`